code/world.dm

```diff
@@ -71,7 +71,8 @@ var/list/map_transition_config = MAP_TRANSITION_CONFIG
 	map_name = "Unknown"
 	#endif
 
-
+	for (var/typepath in (typesof(/datum/menu)-/datum/menu))
```

Comment:
I blame that file being above defines, it's not required to be, really, defines imo should be the highest in the project, so all below can use them.